app/ai/voice/agents/breeze_buddy/managers/calls.py

```
@@ -436,15 +440,25 @@ async def process_backlog_leads():
                 )
-                if blacklist_phone and await is_number_blacklisted(
-                    blacklist_phone, locked_lead.reseller_id
+
+                if customer_phone:
+                    asyncio.create_task(
+                        process_outbound_rate_limit_alert(
+                            customer_phone=customer_phone,
+                            lead_id=str(locked_lead.id),
+                            reseller_id=locked_lead.reseller_id,
+                        )
+                    )
```

Ruff (0.15.7)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)